docs/Using-Pantheon/Accessing-Logs-Using-JSON-RPC.md

```diff
@@ -0,0 +1,164 @@
+# Accessing Logs Using JSON-RPC API Methods
+
+Access logs using JSON-RPC API methods:
+ 
+* [`eth_getFilterChanges`](../Reference/JSON-RPC-API-Methods.md#eth_getfilterchanges)
+* [`eth_getFilterLogs`](../Reference/JSON-RPC-API-Methods.md#eth_getfilterlogs)
+* [`eth_getLogs`](../Reference/JSON-RPC-API-Methods.md#eth_getlogs)
+
+Before using [`eth_getFilterChanges`](../Reference/JSON-RPC-API-Methods.md#eth_getfilterchanges) and 
+[`eth_getFilterLogs`]((../Reference/JSON-RPC-API-Methods.md#eth_getfilterlogs)), create the filter 
+using [`eth_newFilter`](../Reference/JSON-RPC-API-Methods.md#eth_newfilter).
+
+!!! tip
+    The following examples are created using the sample contract included in [Events and Logs](Events-and-Logs.md). 
+
+## Creating a Filter
+
+Create a filter using [`eth_newFilter`](../Reference/JSON-RPC-API-Methods.md#eth_newfilter). 
+
+!!! example
+    
+    If the [example contract](Events-and-Logs.md#example) was deployed to 0x42699a7612a82f1d9c36148af9c77354759b210b, the 
+    following request for `eth_newFilter` creates a filter to log when `valueIndexed` is set to 5. 
+    
+    ```json
+    {
+     "jsonrpc":"2.0",
+     "method":"eth_newFilter",
+     "params":[
+        {
+          "fromBlock":"earliest", 
+          "toBlock":"latest", 
+          "address":"0x42699a7612a82f1d9c36148af9c77354759b210b", 
+          "topics":[
+              ["0xd3610b1c54575b7f4f0dc03d210b8ac55624ae007679b7a928a4f25a709331a8"], 
+              ["0x0000000000000000000000000000000000000000000000000000000000000005"]
+          ]
+         }
+      ],
+      "id":1
+    }
+    ```
+        
+[`eth_newFilter`](../Reference/JSON-RPC-API-Methods.md#eth_newfilter) returns a filter ID hash (for example, `0x1ddf0c00989044e9b41cc0ae40272df3`). 
+
+### Polling Filter for Changes
+
+To poll the filter for changes that have occurred since the last poll, use [`eth_getFilterChanges`](../Reference/JSON-RPC-API-Methods.md#eth_getfilterchanges)
+with the filter ID hash returned by [`eth_newFilter`](../Reference/JSON-RPC-API-Methods.md#eth_newfilter). 
+
+!!! example
```

Comment:
Would it be possible to include the JSON-RPC request for this (and the next) example, using a placeholder for the previously returned filter ID hash--something like:

`{"jsonrpc":"2.0","method":"eth_getFilterChanges","params":["<filter-id-hash>"],"id":1}`

I understand you have to execute a contract to get the result, but I think you could include a request like the one above.

Reply:
I'm not sure there's any point - the request only contains one param which is the filter hash so the example isn't going to be usable (you need to copy the result from the createFilter call). I do link to the reference material so I if they're no clear there's an example there.